isos/bootstrap/bootstrap

```diff
@@ -34,6 +46,27 @@ if mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}; then
 
     cp /bin/tether ${MOUNTPOINT}/.tether/tether
 
+
+    # linux-vdso.so.1 (0x00007ffc94d0d000)
```

Comment:
This is a good question.  We're doing it because `netfilter` doesnt have an appropriate userspace api that one can write against.  All disclaimers say we should use `iptables-restore`.  Kinda dumb, if you ask me.  So we're reduced to copying binaries to make it work in the `initrd`.  I'll add a comment.  Thanks!

PS.  I was harmed.  Does that count?